grpc/src/main/java/com/linecorp/armeria/server/grpc/GrpcServiceBuilder.java

```diff
@@ -651,6 +651,10 @@ public GrpcServiceBuilder unframedGrpcErrorHandler(UnframedGrpcErrorHandler unfr
      *     </li>
      * </ul>
      *
+     * <p>{@link GrpcSerializationFormats#JSON} must be supported via
+     * {@link #supportedSerializationFormats(SerializationFormat...)} to enable HTTP/JSON transcoding.
```

Comment:
The sentence sounds like users always set `GrpcSerializationFormats#JSON` but they don't need it when using the default `GrpcSerializationFormats` which is all `GrpcSerializationFormats#values()`.
How about?
```
When custom {@link #supportedSerializationFormats(SerializationFormat...)} are used,
{@link GrpcSerializationFormats#JSON} must be included to enable HTTP/JSON transcoding.
```

Reply:
Updated 👍